server/src/v2/helpers/cipher.js

```diff
@@ -24,26 +25,37 @@ export function encrypt(body, passphrase, method = 'aes') {
   if (!methods.includes(method))
     return false
 
-  if (method === 'none')
+  if ( method === 'publickey' ) {
+    return cryptoDefault.publicEncrypt(
+      {
+        key: passphrase,
+        oaepHash: "sha256",
```

Comment:
Style. Inconsistent use of single and double quotes. Use single quotes

Reply:
Fixed in 0e48148